apps/api/schedule.py

```diff
@@ -169,6 +170,69 @@ def post(self, proposal_type):
         return [t.id for t in res]
 
 
+class ProposalC3VOCPublishingWebhook(Resource):
+    method_decorators = {"post": [_require_video_api_key]}
+
+    def post(self):
+        if not request.is_json:
+            abort(415)
+
+        payload = request.get_json()
+
+        try:
+            conference = payload["fahrplan"]["conference"]
+            proposal_id = payload["fahrplan"]["id"]
+        except KeyError:
+            abort(422)
+
+        if not payload["is_master"]:
+            # c3voc *should* only send us information about the master
+            # encoding getting published. Aborting early ensures we don't
+            # accidentially delete video information from the database.
+            abort(403)
+
+        if conference != f"emf{event_year()}":
+            abort(422)
+
+        proposal = Proposal.query.get_or_404(proposal_id)
+
+        if payload["voctoweb"]["enabled"]:
+            if payload["voctoweb"]["frontend_url"]:
+                proposal.c3voc_url = payload["voctoweb"]["frontend_url"]
+                proposal.video_recording_lost = False
+            else:
+                # This allows c3voc to notify us if videos got depublished
+                # as well. We do not explicitely set 'video_recording_lost'
+                # here because the video might only need fixing audio or
+                # such.
+                proposal.c3voc_url = ""
```

Comment:
```suggestion
                proposal.c3voc_url = None
```

Reply:
`ProposalResource` sets it to `""` on top of the file :/

Reply:
Ah, drats.  Not a nullable field.  Ok, `""` it is!

Reply:
Mm.. my mistake; in fact it _is_ nullable: https://github.com/emfcamp/Website/blob/888fb8b7d63c7560376bc2d5670b0e46f61fdce0/migrations/versions/9483c0756933_add_proposal_urls.py#L22

I wonder if there's any importance in using `""` instead of `None` there (cc @lukegb?).

Reply:
That seems to be the cause of a bug; I've branched that off into #1771 (not a blocker for this).